src/component/Label.tsx

```diff
@@ -1,15 +1,20 @@
 import React, { cloneElement, isValidElement, ReactNode, ReactElement, createElement, SVGProps } from 'react';
-import isNil from 'lodash/isNil';
-import isFunction from 'lodash/isFunction';
-import isObject from 'lodash/isObject';
 import clsx from 'clsx';
 import { Text } from './Text';
 import { findAllByType, filterProps } from '../util/ReactUtils';
-import { isNumOrStr, isNumber, isPercent, getPercentValue, uniqueId, mathSign } from '../util/DataUtils';
+import {
+  isNumOrStr,
+  isNumber,
+  isPercent,
+  getPercentValue,
+  uniqueId,
+  mathSign,
+  isNullOrUndefined,
+} from '../util/DataUtils';
 import { polarToCartesian } from '../util/PolarUtils';
 import { ViewBox, PolarViewBox, CartesianViewBox, DataKey } from '../util/types';
 
-export type ContentType = ReactElement | ((props: Props) => ReactNode);
+export type ContentType = ReactElement | ((props: Props) => ReactNode) | Function;
```

Comment:
Typescript didn't like this section, as I'm doing `isValidElement(label) || typeof label === 'function'` it can't pass `content={label}` as it can be a function.

I'm not the best with typescript so wasn't sure what to put here. I just pushed with `--no-verify` so I can get some feedback.

Reply:
`((props: Props) => ReactNode)` should be able to handle this. I'd rather not add any `Function` here as that isn't specific enough... I can take a look

Reply:
Yeah, I also thought that, but I get the following ts error when I remove `| Function`.

```
Type 'Function' is not assignable to type 'ContentType'.ts(2322)

Label.tsx(47, 3): The expected type comes from property 'content' which is declared here on type 'IntrinsicAttributes & Omit<SVGProps<SVGTextElement>, "viewBox"> & LabelProps'

(property) LabelProps.content?: ContentType
```

Reply:
You can do something like this

```tsx
  const isLabelContentAFunction = (content: unknown): content is (props: Props) => React.ReactNode => {
    return content && typeof content === 'function';
  };
  
  ...

  if (isLabelContentAFunction(label)) {
    return <Label key="label-implicit" content={label} viewBox={viewBox} />;
  }
```

Reply:
if it is a function we know already that it should match the types we've defined. It just needs asserted upon here